.pre-commit-config.yaml

```diff
@@ -13,6 +13,6 @@ repos:
     hooks:
       - id: black
   - repo: https://github.com/pycqa/isort
-    rev: 5.10.1
```

Comment:
I recall having issues with newer isort and ros2 `.srv` imports in python. not sure why tho, basically meant the sim `reset` service would put it on a line with other imports. Was it causing issues again?

Reply:
It was causing issues with pre-commit - I can't recall the specific error message, but the concensus across multiple github issues was to just update isort (which fixed my issue).